test/recipe/admin/.baked/rich-transient-with-datas.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$type": "Report",
+  "$type": "Record",
   "path": "rich-transient-with-datas",
-  "title": "Rich Transient w/ Data",
+  "title": "Rich Transient w/ Datas",
```

Comment:
this is the page title, no need to use plural. paths are plural because of RESTful conventions, but page title need to be singular.

minor thing, but need to fix to avoid confusion in the future